textproc/man-db/Portfile

```diff
@@ -3,33 +3,34 @@
 PortSystem          1.0
 
 name                man-db
-version             2.8.5
+version             2.8.7
 categories          textproc
 platforms           darwin linux
 license             GPL-3+
 maintainers         {@ylluminarious orbitalimpact.com:georgedp} openmaintainer
 description         Modern, featureful implementation of the Unix man page system.
 homepage            http://man-db.nongnu.org
-master_sites        https://download.savannah.nongnu.org/releases/man-db/
+master_sites        https://quantum-mirror.hu/mirrors/pub/gnusavannah/man-db/
 use_xz              yes
+use_autoreconf      yes
+autoreconf.args     --install --verbose --force
 
 long_description    man-db is an implementation of the standard Unix documentation \
     system accessed using the man command. It uses a Berkeley DB database in place \
     of the traditional flat-text whatis databases. man-db is used by several popular \
     Linux distributions, including: Arch, Debian, Dragora, Fedora, Gentoo, openSUSE, \
     and Ubuntu.
 
-checksums           rmd160   20caab5f9584a27487f117802a3601e7d0a28525 \
-                    sha256   b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1 \
-                    size     1787244
+checksums           rmd160   dd933f76d09159d3e4de25b1efcbec390874d208 \
+                    sha256   b9cd5bb996305d08bfe9e1114edc30b4c97be807093b88af8033ed1cf9beb326 \
+                    size     1839012
 
 depends_lib         port:libpipeline
-depends_build       port:pkgconfig
+depends_build       port:pkgconfig \
```

Comment:
Setting `use_autoreconf yes` should add autoconf and automake (and possibly libtool?) automatically, but using `depends_build` instead of `depends_build-append` probably overwrites this default.

Anyway, it doesn't really matter how the dependency gets specified, the important part is that it is.

Reply:
@neverpanic Ah, ok, I wasn't aware that setting `use_autoreconf yes` would automatically add those dependencies. I'll keep that in mind in the future.